js/wp-seo-term-scraper.js

```diff
@@ -362,5 +374,8 @@
 		jQuery( window ).trigger( 'YoastSEO:ready' );
 
 		termScraper.initKeywordTabTemplate();
+
+		//init Plugins
+		window.yoastReplaceVarPlugin = new YoastReplaceVarPlugin();
```

Comment:
Is there a reason to expose this globally? I want to reduce this as much as possible until we really nailed our public API.